include/lsst/afw/geom/SpherePoint.h

```diff
@@ -98,6 +98,9 @@ class SpherePoint final {
      * @throws pex::exceptions::InvalidParameterError
      *         Thrown if `vector` is the zero vector.
      *
+     * @note If the SpherePoint is at a pole then longitude is set to 0.
+     * That provides predictable behavior for methods such as bearingTo and offset.
+     *
```

Comment:
Are there any guarantees for what `northPole.getLongitude()` or `southPole.getLongitude()` returns, or does the pole's longitude depend on the arguments to `SpherePoint(lon, lat)`?

Reply:
IIRC `bearingTo` will auto-generate a link, but `offset` will not. Have you checked?

Reply:
I'll add `@ref`